sdks/builds/maccat.mk

```
@@ -0,0 +1,128 @@
+
+maccat_BIN_DIR = $(TOP)/sdks/out/maccat-bin
+maccat_PKG_CONFIG_DIR = $(TOP)/sdks/out/maccat-pkgconfig
+maccat_LIBS_DIR = $(TOP)/sdks/out/maccat-libs
+maccat_TPN_DIR = $(TOP)/sdks/out/maccat-tpn
+maccat_MONO_VERSION = $(TOP)/sdks/out/maccat-mono-version.txt
+
+maccat_ARCHIVE += maccat-bin maccat-pkgconfig maccat-libs maccat-tpn maccat-mono-version.txt
+ADDITIONAL_PACKAGE_DEPS += $(maccat_BIN_DIR) $(maccat_PKG_CONFIG_DIR) $(maccat_LIBS_DIR) $(maccat_TPN_DIR) $(maccat_MONO_VERSION)
+
+##
+# Parameters
+#  $(1): target
+#  $(2): host arch
+#  $(3): xcode dir
+define MacCatTemplate
+
+maccat_$(1)_PLATFORM_BIN=$(3)/Toolchains/XcodeDefault.xctoolchain/usr/bin
+
+#
+# HACK: fak: The -target is placed in the CC define per the recommendation of
+# libtool who acknowledge that some parameters are just not passed through
+# to the compiler. You can use -Wc, flags, but I failed to get the working appropriately.
+#
+_maccat-$(1)_CC=$$(CCACHE) $$(maccat_$(1)_PLATFORM_BIN)/clang -target $(2)-apple-ios13.0-macabi
+_maccat-$(1)_CXX=$$(CCACHE) $$(maccat_$(1)_PLATFORM_BIN)/clang++ -target $(2)-apple-ios13.0-macabi
+
+_maccat-$(1)_AC_VARS= \
+	ac_cv_func_system=no \
+	ac_cv_c_bigendian=no \
+	ac_cv_func_fstatat=no \
+	ac_cv_func_readlinkat=no \
+	ac_cv_func_getpwuid_r=no \
+	ac_cv_func_posix_getpwuid_r=yes \
+	ac_cv_header_curses_h=no \
+	ac_cv_header_localcharset_h=no \
+	ac_cv_header_sys_user_h=no \
+	ac_cv_func_getentropy=no \
+	ac_cv_func_futimens=no \
+	ac_cv_func_utimensat=no \
+	ac_cv_func_shm_open_working_with_mmap=no \
+	mono_cv_sizeof_sunpath=104
+
+_maccat-$(1)_CFLAGS= \
+	$$(maccat-$(1)_SYSROOT) \
+	-fexceptions
+
+_maccat-$(1)_CXXFLAGS= \
+	$$(maccat-$(1)_SYSROOT)
+
+_maccat-$(1)_CPPFLAGS= \
+	-DSMALL_CONFIG -D_XOPEN_SOURCE -DHOST_IOS -DHOST_MACCAT -DHAVE_LARGE_FILE_SUPPORT=1
+
+_maccat-$(1)_LDFLAGS= \
+	-target $(2)-apple-ios13.6-macabi \
```

```suggestion
	-target $(2)-apple-ios13.0-macabi \
```

It'd be nice to extract this version into a variable in sdks/versions.mk, e.g. `MACCAT_IOS_VERSION_MIN=13.0`

---

I will add `MACCAT_IOS_VERSION_MIN`, thanks!

---

Fixed in https://github.com/mono/mono/pull/20537/commits/691a92290a4f78b06c6c0acb275b514aba0b4fd3